virttest/shared/cfg/guest-os/Linux/CentOS-Stream.cfg

```diff
@@ -0,0 +1,22 @@
+- CentOS-Stream:
+    no setup
+    shell_prompt = "^\[.*\][\#\$]\s*$"
+    nic_hotplug, multi_nics_hotplug:
+        modprobe_module = acpiphp
+    block_hotplug:
+        modprobe_module = acpiphp
+    unattended_install, check_block_size..extra_cdrom_ks,svirt_install, with_installation, blockdev_commit_install, blockdev_snapshot_install:
+        wait_no_ack = yes
```

Comment:
This is missing on the RHEL 9 definition, causing unattended_install tests to fail with `VM is dead`:

```
[stdlog] 2023-02-20 19:17:47,448 avocado.virttest.env_process DEBUG| Thread quit. Used to failed to get registers info from guest 20230220-184629-k3HR4PHB for 2 times.
[stderr] Traceback (most recent call last):
[stderr]   File "/home/cleber/src/avocado-vt/virttest/qemu_vm.py", line 302, in verify_alive
[stderr]     virt_vm.BaseVM.verify_alive(self)
[stderr]   File "/home/cleber/src/avocado-vt/virttest/virt_vm.py", line 700, in verify_alive
[stderr]     raise VMDeadError
[stderr] virttest.virt_vm.VMDeadError: VM is dead
[stderr] 
[stderr] During handling of the above exception, another exception occurred:
[stderr] 
[stderr] Traceback (most recent call last):
[stderr]   File "/home/cleber/src/avocado-vt/avocado_vt/plugins/vt_runner.py", line 69, in runTest
[stderr]     raise self.__status  # pylint: disable-msg=E0702
[stderr]     ^^^^^^^^^^^^^^^^^^^
[stderr]   File "/home/cleber/src/avocado-vt/avocado_vt/test.py", line 145, in setUp
[stderr]     self._runTest()
[stderr]   File "/home/cleber/src/avocado-vt/avocado_vt/test.py", line 263, in _runTest
[stderr]     run_func(self, params, env)
[stderr]   File "/home/cleber/avocado/data/avocado-vt/virttest/test-providers.d/downloads/io-github-autotest-qemu/generic/tests/unattended_install.py", line 24, in run
[stderr]     unattended_install.run(test, params, env)
[stderr]   File "/home/cleber/src/avocado-vt/virttest/error_context.py", line 135, in new_fn
[stderr]     return fn(*args, **kwargs)
[stderr]            ^^^^^^^^^^^^^^^^^^^
[stderr]   File "/home/cleber/src/avocado-vt/virttest/tests/unattended_install.py", line 1432, in run
[stderr]     raise e
[stderr]   File "/home/cleber/src/avocado-vt/virttest/tests/unattended_install.py", line 1388, in run
[stderr]     vm.verify_alive()
[stderr]   File "/home/cleber/src/avocado-vt/virttest/qemu_vm.py", line 306, in verify_alive
[stderr]     raise virt_vm.VMDeadError(self.process.get_status(),
[stderr] virttest.virt_vm.VMDeadError: VM is dead
```